stock_picking_invoicing/__openerp__.py

```diff
@@ -29,5 +29,5 @@
     "data": [
         "stock_view.xml",
     ],
-    "installable": True
+    "installable": False
```

Comment:
Hum, guess I forgot this after doing some test interaction isolation to restore green status.